src/core/memory.d

```diff
@@ -809,3 +809,41 @@ struct GC
         gc_runFinalizers( segment );
     }
 }
+
+/// Pure variant of `malloc` that doesn't affect global variable `errno`.
```

Comment:
The teleological description is better than the description in terms of implementation details. I.e. if a platform offered access to an errno-less malloc (as e.g. would be easy to hack with jemalloc), pureMalloc would use that.